smk/assembly.smk

```diff
@@ -237,7 +237,9 @@ rule correct_spadeshammer:
         """
         mkdir -p $(dirname {output.fwd})
         time (
-            {spades_script} --only-error-correction -1 {input.reads[0]} -2 {input.reads[1]} -t {threads} -m {resources.mem} -o {wildcards.run} --phred-offset {params.qoffset}
+            cp {input.reads[0]} {wildcards.run}.1.fq.gz
```

Comment:
Could we switch these two 'cp's to rsync to ensure there is no failure and checksum is verified?